keyRegistration/js/registration.js

```diff
@@ -144,7 +144,7 @@ $(document).ready(function()
         }
     });
 
-    function resetFields(){
+    function resetFields() {
```

Comment:
Opening brace should be on a new line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*